packages/ui/src/views/repo/pull-request/compare/components/pull-request-compare-diff-list.tsx

```diff
@@ -115,10 +122,44 @@ const PullRequestCompareDiffList: FC<PullRequestCompareDiffListProps> = ({
     <>
       <ListActions.Root>
         <ListActions.Left>
-          <p className="text-14 leading-tight text-foreground-4">
-            Showing <span className="text-foreground-accent">{diffStats.files_changed || 0} changed files </span>
-            with {diffStats.additions || 0} additions and {diffStats.deletions || 0} deletions
-          </p>
+          <DropdownMenu>
+            <p className="text-14 leading-tight text-foreground-4">
+              Showing{' '}
+              <DropdownMenuTrigger asChild>
+                <span className="text-foreground-accent cursor-pointer ease-in-out">
+                  {diffStats.files_changed || 0} changed files
+                </span>
+              </DropdownMenuTrigger>{' '}
+              with {diffStats.additions || 0} additions and {diffStats.deletions || 0} deletions
+            </p>
+            <DropdownMenuContent align="end">
+              <DropdownMenuGroup>
+                {diffData?.map(diff => (
+                  <DropdownMenuItem
+                    key={diff.filePath}
+                    onClick={() => {}}
+                    className="flex w-80 items-center justify-between cursor-pointer px-3 py-2"
+                  >
+                    <Text size={1} className="flex-1 overflow-hidden truncate text-primary">
```

Comment:
```suggestion
                    <Text size={1} className="flex-1 overflow-hidden truncate text-primary">
```

Whenever possible, we should avoid using the `Text` component, as the plan is to completely phase it out in the future. Instead, let’s use the appropriate HTML tag directly

Reply:
sure